test/ExecutionFramework.h

```diff
@@ -76,7 +85,13 @@ class ExecutionFramework
 		std::map<std::string, Address> const& _libraryAddresses = std::map<std::string, Address>()
 	)
 	{
-		compileAndRunWithoutCheck(_sourceCode, _value, _contractName, _arguments, _libraryAddresses);
+		multiSourceCompileAndRunWithoutCheck(
+			std::map<std::string, std::string>({{ "", _sourceCode }}),
```

Comment:
```suggestion
			std::map<std::string, std::string>{{"", _sourceCode}},
```
I think this should work too:
```suggestion
			{{"", _sourceCode}},
```

Reply:
How about the `{{"", _sourceCode}}` form? I see you used it in other places but here you're still using `std::map<std::string, std::string>{...}`